### PR TITLE
fix migration to reset all slugs

### PR DESCRIPTION
**Pull request by rgraber**

### CODE REVIEW

File: course_discovery/apps/course_metadata/migrations/0199_url_slug_history.py [hunk 1]
```diff
@@ -16,35 +16,22 @@ class Migration(migrations.Migration):
     ]
 
     def migrate_data_forward(apps, schema_editor):
-        updated_draft_course_pks = []
+        published_drafts = []
```
Comment (mikix) [inline course_discovery/apps/course_metadata/migrations/0199_url_slug_history.py:19]: nit: a set would make more sense for this use case.

File: course_discovery/apps/course_metadata/migrations/0199_url_slug_history.py [hunk 2]
```diff
@@ -16,35 +16,22 @@ class Migration(migrations.Migration):
...
         Course = apps.get_model('course_metadata', 'course')
         CourseUrlSlug = apps.get_model('course_metadata', 'courseUrlSlug')
         for instance in Course.everything.all().order_by('draft'):
```
Comment (mikix) [inline course_discovery/apps/course_metadata/migrations/0199_url_slug_history.py:22]: Not part of this change I know, but order by draft? Because you are collecting the drafts up in this list?

File: course_discovery/apps/course_metadata/migrations/0199_url_slug_history.py [hunk 3]
```diff
@@ -16,35 +16,22 @@ class Migration(migrations.Migration):
...
                 historical_slug = CourseUrlSlug.objects.create(
                     course=instance,
                     partner=instance.partner,
                     is_active=True,
-                    is_active_on_draft=current_slug_is_active_on_draft
+                    is_active_on_draft=True
                 )
-                # need to set slug after create so it's not overridden with the autoslugification
-                historical_slug.url_slug = instance.url_slug
                 historical_slug.save()
```
Comment (mikix) [inline course_discovery/apps/course_metadata/migrations/0199_url_slug_history.py:31]: Can drop this line.

**Review by mikix — APPROVED** [review]
Sure? I'm less familiar with whether the logic changes make sense. But assume it's great.

File: course_discovery/apps/course_metadata/migrations/0199_url_slug_history.py (hunk 2 above)
Comment (rgraber) [reply course_discovery/apps/course_metadata/migrations/0199_url_slug_history.py:22]: yep. the goal is to only manually update drafts that don't have associated published courses
